_includes/api/en/4x/req-fresh.md

```diff
@@ -2,12 +2,19 @@
 
 When the response is still "fresh" in the client's cache `true` is returned, otherwise `false` is returned to indicate that the client cache is now stale and the full response should be sent.
 
+In order to check whether the response is still "fresh" in the client's cache or not, two tags will be used ETag and If-None-Match. ETag will be fetched from response (saved internally from application) and other tag will be received from client's request.
+
 When a client sends the `Cache-Control: no-cache` request header to indicate an end-to-end reload request, this module will return `false` to make handling these requests transparent.
 
 Further details for how cache validation works can be found in the
 [HTTP/1.1 Caching Specification](https://tools.ietf.org/html/rfc7234).
 
 ```js
+
+// Get value for ETag where you saved it initially, In general you 
+// may use a file to store timestamps with API request name
+
+res.set('ETag', "foo")
```

Comment:
The example that existed wasn't very clear to begin with. Setting an etag here doesn't mean that req.fresh will be true, so the example gets less clear with the addition.

It's easier perhaps to demonstrate that setting a new, highly random etag on every request can guarantee that req.fresh is false. That way we create an example which will always have the demonstrated output.